packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -93,7 +93,7 @@ export default class WalletService {
   constructor(storePath: string = defaultStorePath) {
     this.storePath = `${app.getPath('userData')}/${storePath}`
     fs.mkdirSync(this.storePath, { recursive: true })
-    this.listStore = new Store(this.storePath, 'wallets.json')
+    this.listStore = new Store('wallet', 'wallets.json')
```

Comment:
Also need to take care of `this.storePath`, which will be used to store keystore files. `wallets.json` and all keystore files should be stored int he same `wallets` (not `wallet`) folder.

Reply:
fixed in https://github.com/nervosnetwork/neuron/pull/329/commits/fdbaf47ab5f7e277dbeb645e2808227217ac8b18

Reply:
On LN 94 `this.storePath` was constructed with the tased in param. Here we need to do to things:

* Support `test/` env and default folder location as I described in the other comment.
* Remove the `storePath` param so wallet list and keystore files use the same `wallets` folder.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/329/commits/5c18936eac71753db64f1b4bcf35449c970323d1 and https://github.com/nervosnetwork/neuron/pull/329/commits/d69b0c2f5b50b169de76e2d5ab7a67dcd34a6ce7, use NODE_EVN=test for testing which will create /test/ dir in the store dir